Book-A-Desk.Api/EmailNotification.fs

```diff
@@ -0,0 +1,46 @@
+namespace Book_A_Desk.Api
```

Comment:
So, I think that this EmailNotification should belong in another project, similar to how the DynamoDb code is in .Infrastructure, because it doesn't exist for the API, but to provide infrastructure code to the Handlers. Two suggestions:

1. Put it in .Infrastructure
2. Create a new project, maybe .Facade (or any other semantically meaningful name for external services like these) and put it in there instead.

Reply:
Yes - I fully agree. It should go somewhere else in the future. The idea with this PR is to have a simple implementation for the MVP to be used earlier the expected.

Reply:
Good with me!